phpstan.neon.dist

```diff
@@ -1,7 +1,20 @@
 parameters:
     level: 2
     paths:
-        - redaxo/src/
+        - redaxo/src/core
```

Comment:
Warum muss die liste hier geändert werden? Fehlen hier bewusst jetzt addons?

Reply:
Ich wüsste jetzt nicht, dass welche fehlen. 

Aber so werden wirklich nur die addons aus diesem repo genommen wenn es es lokal ausführt (z.b. für generierung der baseline)

Reply:
Ahh macht sinn.. dann am besten nen kommentar in der config (Geht mit #)